config/initializers/rails_performance.rb

```diff
@@ -0,0 +1,8 @@
+if defined?(RailsPerformance)
+  RailsPerformance.setup do |config|
+    config.redis    = Redis.new(url: ENV.fetch('REDIS_PERFORMANCE_URL') { 'redis://localhost:6379/1' })
+    config.duration = 4.hours
+    config.enabled  = true
+    config.http_basic_authentication_enabled = false
```

Comment:
don't we need to have this enabled so that only admins access the performance panel?

Reply:
Sure, do i hardcode the credentials aswell as credentials for activeadmin? Or do you preffer to save them in the ENV and use the same for both?

Reply:
maybe env variables is better, or if you can authorize by `AdminUser` then better

Reply:
not sure if it's possible with this gem

Reply:
Ok, I will check, otherwise I use ENV